frontend/src/components/GSelectCloudProfile.vue

```diff
@@ -43,36 +43,44 @@ import { withFieldName } from '@/utils/validators'
 import find from 'lodash/find'
 
 const props = defineProps({
-  modelValue: { // cloudProfileRef
+  modelValue: {
     type: Object,
     default: null,
   },
+  cloudProfiles: {
+    type: Array,
+    required: true,
+  },
+  namespacedCloudProfiles: {
+    type: Array,
+    required: false,
+    default: () => [],
+  },
 })
 
-const cloudProfileRef = toRef(props.modelValue)
+const cloudProfileRef = toRef(props, 'modelValue')
+const cloudProfilesRef = toRef(props, 'cloudProfiles')
+const namespacedCloudProfilesRef = toRef(props, 'namespacedCloudProfiles')
```

Comment:
```suggestion
const cloudProfiles = toRef(props, 'cloudProfiles')
const namespacedCloudProfiles = toRef(props, 'namespacedCloudProfiles')
```

```js
const cloudProfileRef = toRef(props, 'modelValue')
```
this we keep as cloudProfile"Ref" (ref in this case has nothing to do with vue refs)